plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -578,6 +578,12 @@ public OptionalInt getMaxWriteParallelism(ConnectorSession session)
         return delegate.getMaxWriteParallelism(session);
     }
 
+    @Override
+    public OptionalInt getMaxColumnNameLength(ConnectorSession session)
```

Comment:
it's not only column names but generally identifiers (alias has limit too).

`getMaxIdentifierNameLength`?

Reply:
I renamed it because that's what `DatabaseMetaData` calls the field.